src/patch-events.js

```diff
@@ -275,6 +275,10 @@ export function addEventListener(type, fn, optionsOrCapture) {
           return;
         }
       }
+      if(fn.handleEvent) {
```

Comment:
The `this` value in an EventTarget is the object containing the handleEvent function, not the element from which the event derives.